index.html

```diff
@@ -3,87 +3,82 @@
 	
 	<head>
 		<title>Charlotte Buckley - Portfolio</title>
-		<link rel="stylesheet" type ="text/css" href="style.css"> 
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
 		<link rel="stylesheet" type="text/css" href="normalize.css">
+		<link rel="stylesheet" type ="text/css" href="style.css"> 
 		<link rel="preconnect" href="https://fonts.googleapis.com">
 		<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
 		<link href="https://fonts.googleapis.com/css2?family=Varela+Round&display=swap" rel="stylesheet">
 	</head>	
 	
 	<body>		
-		<nav class="nav-bar">
+		<nav class="nav-bar" id="top">
```

Comment:
you could be more semantic with the naming of this, this is describing where the element _is_ not what it `does`, consider something like 'nav-bar'